site/src/main/mdoc/alternative_effects.md

```diff
@@ -71,7 +73,7 @@ val data: String = program[IO].unsafeRunSync()
 
 ### monix.eval.Task _(neotypes-monix)_
 
-```scala mdoc:compile-only
```

Comment:
Since there are two incompatible versions of cats-effect in the classpath the snippet cannot be compiled. 
Thus I've temporarily disabled compilation for this and further snippets.